lib/common.sh

```diff
@@ -149,6 +149,9 @@ else
   IPAMBRANCH="${IPAMBRANCH:-main}"
 fi
 
+export BMOREPO="https://github.com/Nordix/baremetal-operator.git"
```

Comment:
Will you remove these after the test is done? (I assume yes, but just in case)

Reply:
Yes indeed! There is a separate commit for testing that I will drop once reviewers are happy :slightly_smiling_face: